components/com_contact/tmpl/contact/default.php

```diff
@@ -62,7 +63,7 @@
 		<?php $contactLink = RouteHelper::getCategoryRoute($this->item->catid, $this->item->language); ?>
 		<h3>
 			<span class="contact-category"><a href="<?php echo $contactLink; ?>">
-				<?php echo $this->escape($this->item->category_title); ?></a>
+					<?php echo $this->escape($this->item->category_title); ?></a>
```

Comment:
```suggestion
				<?php echo $this->escape($this->item->category_title); ?></a>
```
Revert wrong change on indentation.